lib/workers/repository/onboarding/branch/index.spec.ts

```diff
@@ -369,61 +395,46 @@ describe('workers/repository/onboarding/branch/index', () => {
         config.onboardingRebaseCheckbox = true;
         OnboardingState.prUpdateRequested = false;
         scm.getFileList.mockResolvedValueOnce(['package.json']);
+        onboardingCache.isOnboardingBranchModified.mockResolvedValueOnce(true);
         platform.findPr.mockResolvedValueOnce(null);
       });
 
       it('detects unsupported platfom', async () => {
         const pl = 'bitbucket';
         GlobalConfig.set({ platform: pl });
         platform.getBranchPr.mockResolvedValueOnce(mock<Pr>({}));
-
         await checkOnboardingBranch(config);
-
         expect(logger.trace).toHaveBeenCalledWith(
           `Platform '${pl}' does not support extended markdown`
         );
         expect(OnboardingState.prUpdateRequested).toBeTrue();
-        expect(git.mergeBranch).toHaveBeenCalledOnce();
-        expect(scm.commitAndPush).toHaveBeenCalledTimes(0);
       });
 
       it('detects missing rebase checkbox', async () => {
         const pr = { bodyStruct: { rebaseRequested: undefined } };
         platform.getBranchPr.mockResolvedValueOnce(mock<Pr>(pr));
-
         await checkOnboardingBranch(config);
-
         expect(logger.debug).toHaveBeenCalledWith(
           `No rebase checkbox was found in the onboarding PR`
         );
         expect(OnboardingState.prUpdateRequested).toBeTrue();
-        expect(git.mergeBranch).toHaveBeenCalledOnce();
-        expect(scm.commitAndPush).toHaveBeenCalledTimes(0);
       });
 
       it('detects manual pr update requested', async () => {
         const pr = { bodyStruct: { rebaseRequested: true } };
         platform.getBranchPr.mockResolvedValueOnce(mock<Pr>(pr));
-
         await checkOnboardingBranch(config);
-
         expect(logger.debug).toHaveBeenCalledWith(
           `Manual onboarding PR update requested`
         );
         expect(OnboardingState.prUpdateRequested).toBeTrue();
-        expect(git.mergeBranch).toHaveBeenCalledOnce();
-        expect(scm.commitAndPush).toHaveBeenCalledTimes(0);
       });
 
       it('handles unchecked rebase checkbox', async () => {
         const pr = { bodyStruct: { rebaseRequested: false } };
         platform.getBranchPr.mockResolvedValueOnce(mock<Pr>(pr));
-
         await checkOnboardingBranch(config);
-
         expect(OnboardingState.prUpdateRequested).toBeFalse();
-        expect(git.mergeBranch).toHaveBeenCalledOnce();
-        expect(scm.commitAndPush).toHaveBeenCalledTimes(0);
```

Comment:
why is this removed? especially the last is a good test to not accidentally commit.

Reply:
forgot to revert after testing ):